github/store-github.sh

```diff
@@ -47,3 +48,16 @@ resinci-deploy store github ${files} \
   --head_owner=${headOwner} \
   --head_repo=${headRepo} \
   --head_branch=${headBranch}
+
+# PR the contract file to the platform repo
+contract=$(find . -type f \( -iname balena.yml -o -iname balena.cue \) -exec basename {} \;)
+
+if [ "${contract}" == "" ]; then
+  exit 1
```

Comment:
This should be exit 0 as we want versioning to continue even if the contract is not there

Reply:
Oops, forgot about that one, done.